src/client-lib/api.js

```diff
@@ -0,0 +1,44 @@
+import fetch from 'isomorphic-fetch';
+
+const API_PREFIX = '/api/v0';
+
+function makeRequest(uri, options) {
+    let res;
+    return fetch(uri, options)
+    .then(response => {
+        res = response;
+        return res.json();
+    })
+    .then(body => {
+        if (res.status >= 200 && res.status < 300) {
```

Comment:
Can you check this up on line 9 to avoid the awkward res variable and extra .json()?